### ISSUE TITLE
Migrate to Python3 ?

### ISSUE THREAD

**Issue by SillySam** [Griffintaur/News-At-Command-Line#26]
This would be great in Python 3, I think this would be appreciated.

**Comment by Griffintaur** [comment]
cool. Go ahead

**Comment by SillySam** [comment]
@Griffintaur I have submitted a [PR](https://github.com/Griffintaur/News-At-Command-Line/pull/27), do you think you can take a look at it, thanks

**Comment by SillySam** [comment]
@Griffintaur depending on what you want, you might want to think about having the Python 2.7 version in a separate branch for those that don't use Python 3.

**Comment by TomFaulkner** [comment]
Is this going to get merged? I noticed there are several unmerged PRs and master hasn't been updated in months.

**Comment by SillySam** [comment]
I know right, this is a little frustrating. If it is not merged after a week I'll make a separate repo.

**Comment by TomFaulkner** [comment]
I was thinking of doing the same.

Aside from Python 3, I saw you did some cleanup, did you do anything with
filenames and pathing?

**Comment by SillySam** [comment]
@TomFaulkner I have not changed file names/paths. If it helps, you can fork my copy: https://github.com/SillySam/News-At-Command-Line

I will be happy to contribute to your copy as this guy doesn't seem to active.

PS: I am just about to make another change on my fork that removes the owner's file headers and email, etc.

**Comment by SillySam** [comment]
@TomFaulkner I also want to note, when you make your own copy, please make your API key as I don't think the owner would like the key going viral :D

**Comment by TomFaulkner** [comment]
Yeah, that is what I had in mind. Since moving to Python 3, without maintaining 2.7 compatibility is a breaking change, I think it would be a good time to make other breaking changes for PEP8 and a more pythonic ways of doing things.

**Comment by TomFaulkner** [comment]
Posted this as a PR on @SillySam's fork. But, thought I would mention it here. I have a more Pythonic fork from @SillySam's fork. I cleaned up quite a bit, more still needs done, but it's posted at least. I plan to do more as I get time.

https://github.com/TomFaulkner/News-At-Command-Line

**Comment by TomFaulkner** [comment]
Since this was merged, I moved discussion to #30 so this issue can be closed.
